update-agent/t/orb_update_agent.conf

```diff
@@ -3,8 +3,8 @@ components = "/usr/persistent/components.json"
 cacert = "/etc/ssl/worldcoin-staging-ota.pem"
 clientkey = "/etc/ssl/private/worldcoin-staging-ota-identity.key"
 update_location = "/mnt/claim.json"
-workspace = "/mnt/updates"
-downloads = "/mnt/updates/downloads"
+workspace = "/mnt/scratch"
```

Comment:
will this be ok for pearl too?

Reply:
ah wait, what is 't' directory? is it tests? Should it be just named tests? why it is named like this